mobile/app/build.gradle

```
@@ -16,16 +16,16 @@ def getVersionCodeTimeStamps = { ->
     return Integer.parseInt(unixtime)
 }
 
-def metaFDroid(outputFileName) {
-    new File(outputFileName, "version_fdroid.txt").text = [
-            ["versionName", android.defaultConfig.versionName].join("="),
-            ["versionCode", android.defaultConfig.versionCode].join("="),
+def metaFDroid(File apkDir, variant) {
+    File outputFile = new File(apkDir, "version_fdroid.txt")
+    outputFile.text = [
```

Maybe we could use the js string template instead of array joining - https://www.w3schools.com/js/js_string_templates.asp

```suggestion
    outputFile.text = 
`versionName=${...}
versionCode=${...}`
```